django_migrations_ci/management/commands/migrateci.py

```diff
@@ -71,8 +71,8 @@ def handle(
         elif parallel is not None:
             parallel = int(parallel)
 
-        storage_class = get_storage_class(storage_class)
-        default_storage_class = get_storage_class(
+        storage_class = import_string(storage_class or settings.DEFAULT_FILE_STORAGE)
```

Comment:
`DEFAULT_FILE_STORAGE` was removed too, maybe this default value is not necessary.